openml/extensions/sklearn/extension.py

```diff
@@ -1029,11 +1031,15 @@ def _run_model_on_fold(
         self,
         model: Any,
         task: 'OpenMLTask',
+        X_train: Union[np.ndarray, scipy.sparse.spmatrix, pd.DataFrame],
+        y_train: np.ndarray,
         rep_no: int,
```

Comment:
can be removed, together with `fold_no`, `sample_no`

Reply:
`rep_no` and `fold_no` are necessary to create trace objects.